rolling/distribution.yaml

```diff
@@ -1516,6 +1516,17 @@ repositories:
       url: https://github.com/ros/pluginlib.git
       version: ros2
     status: maintained
+  point_cloud_msg_wrapper:
+    release:
+      tags:
+        release: release/rolling/{package}/{version}
+      url: https://gitlab.com/ApexAI/point_cloud_msg_wrapper-release
+      version: 1.0.5-1
+    source:
+      type: git
+      url: https://gitlab.com/ApexAI/point_cloud_msg_wrapper
+      version: rolling
+    status: developed
```

Comment:
@carlossvg no `doc` entry?

Reply:
@hidmic I added a doc entry in 692eac57544b396b0961642e82b304744ed3301c, thanks for pointing this out.